Frontend/src/app/components/footer/footer.component.spec.ts

```diff
@@ -1,18 +1,18 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {FooterComponent} from './footer.component';
-import {RouterTestingModule} from "@angular/router/testing";
-import {AppComponent} from "../../app.component";
-import {ImprintComponent} from "../../pages/imprint/imprint.component";
-import {ActivatedRoute, Router} from "@angular/router";
+import {RouterTestingModule} from '@angular/router/testing';
+import {AppComponent} from '../../app.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
import {AppComponent} from "../../app.component";
```